### PR TITLE
Make use of the status to display info

### LINKED ISSUE #44: Make use of status
The bot should display it's prefix and other info in it's status

### PR DESCRIPTION
**Pull request by greenbigfrog**
Not sure if this is the final text I want to use, but this will do for now.

Closes #44

### CODE REVIEW

File: src/discordtipbot/discordbot.cr [hunk 1]
```diff
@@ -152,6 +152,14 @@ class DiscordBot
         end
       end
     end
+
+    # Make use of the status to display info
+    spawn do
+      sleep 2
```
Comment (z64) [inline src/discordtipbot/discordbot.cr]: ?

Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: If immediately used, the discord bot isn't authenticated yet, and discord closes the connection

Comment (z64) [reply src/discordtipbot/discordbot.cr]: This fiber should be spawned from READY then.

File: src/discordtipbot/discordbot.cr [hunk 2]
```diff
@@ -152,6 +152,14 @@ class DiscordBot
...
+      Discord.every(10.seconds) do
```
Comment (langerhans) [inline src/discordtipbot/discordbot.cr]: The rate limit for presence changes is 12 seconds. See just below here: https://discordapp.com/developers/docs/topics/gateway#activity-object-example-activity-with-rich-presence

Comment (z64) [reply src/discordtipbot/discordbot.cr]: This data is probably pretty unlikely to show meaningful change at that rate anyways. I would back this off even further to `1.minute`. No reason to fill the RL bucket that quickly, when there's other things we may need to be sending over the websocket as well (heartbeats, etc)

File: src/discordtipbot/discordbot.cr (hunk 1 above)
Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: Good idea 🤔

File: src/discordtipbot/discordbot.cr (hunk 2 above)
Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: Oops. I thought I had changed it to once per minute before committing

**Comment by greenbigfrog** [comment]
I guess I did forget to squash...

**Pull request merged**